build/yaml/templates/npm-test-steps.yml

```diff
@@ -0,0 +1,26 @@
+steps:
+# You are probably wondering "Why not just run 'yarn exec npm run --if-present test'?".
```

Comment:
Huh - strange! I thought it would work but I guess the environment isn't consistent with running `yarn SCRIPT_NAME`.

Reply:
I couldn't find much helpful documentation that explained the difference in the environmental setup unfortunately. If anyone finds anything of note, let me know and I can refactor.